src/App/App.js

```diff
@@ -14,15 +15,41 @@ class App extends Component {
   }
 
   componentDidMount() {
-    getArtists().then(data => console.log(data))
-    console.log('hello');
+    this.getRandomArtist()
   }
 
 
+  getRandomArtist() {
+    let artistAmount = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14, 15]
+    let artistId = artistAmount.map((num) => {
+      let radomNumber = Math.random() * (300 - num) + num;
+      return Math.round(radomNumber)
+    })
+
+    let fetchArtists = artistId.map((id) => {
+      return getArtists(id).then((data) => {
+        if (data.images.length) {
+          return this.setState(state => state.artists.push(data))
+        } else {
+          console.log('no image');
+        }
+      })
+        .catch((error) => {
+          console.log(error)
+        });
+    })
+    
+  }
+
+  
+
   render() {
     return (
-      <div>
-        <LandingPage/>
+      <div className="main-section">
+        <Routes>
+          <Route path='/' element={<LandingPage artistIds={this.state.artistIds} fetchArtists={this.fetchArtists} />} exact />
```

Comment:
On Line 50, you're passing down this.state.artistIds to Landing page but I don't where you have that property stored in state. I see that you aren't using it in your LandingPage component so maybe it's left over from older code.